client/my-sites/marketing/main.jsx

```diff
@@ -105,7 +133,13 @@ export const Sharing = ( {
 				brandFont
 				className="marketing__page-heading"
 				headerText={ titleHeader }
-				subHeaderText={ description }
+				subHeaderText={ get(
+					selected,
+					'description',
+					translate(
+						'Explore tools to build your audience, market your site, and engage your visitors.'
+					)
+				) }
```

Comment:
This is perfect in the context of this file and can definitely stay. But in the future, if there is a native alternative like the following:
`selected?.description ?? translate(
						'Explore tools to build your audience, market your site, and engage your visitors.'`
						
I recommend using it. `get` is like 1KB. `? + ??` is 3 bytes.

Reply:
Thanks, removed it